core/src/main/java/io/el/concurrent/SingleThreadEventLoop.java

```diff
@@ -151,7 +152,7 @@ private void start() {
       success = true;
     } finally {
       if (!success) {
-        stateUpdater.compareAndSet(this, State.STARTED, State.NOT_STARTED);
+        stateUpdater.compareAndSet(this, State.STARTED, State.TERMINATED);
```

Comment:
If event loop is failed to start it is not in `TERMINATED` state but in `NOT_STARTED` state. plz revert this change